src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -442,31 +457,42 @@ def show(
     ) -> Tuple[np.ndarray, np.ndarray]:
         """Display the aggregated point cloud."""
         if backend == "open3d":
-            return self._show_open3d(**backend_kwargs)
+            return self._show_open3d(instances, **backend_kwargs)
         elif backend == "pytorch3d":
-            return self._show_pytorch3d(**backend_kwargs)
+            return self._show_pytorch3d(instances, **backend_kwargs)
         else:
             raise NotImplementedError(
                 f"Uknown backend {backend}, must be 'open3d' or 'pytorch3d"
             )
 
-    def _show_pytorch3d(self, **plot_scene_kwargs):
+    def _show_pytorch3d(self, instances: bool = True, **plot_scene_kwargs):
         from pytorch3d.vis.plotly_vis import AxisArgs, plot_scene
 
         from home_robot.utils.bboxes_3d_plotly import plot_scene_with_bboxes
 
         points, _, _, rgb = self.voxel_pcd.get_pointcloud()
 
+        traces = {}
+
         # Show points
         ptc = Pointclouds(points=[points], features=[rgb])
+        traces["Points"] = ptc
 
         # Show instances
-        bounds, names = zip(*[(v.bounds, v.category_id) for v in self.get_instances()])
-        detected_boxes = BBoxes3D(
-            bounds=[torch.stack(bounds, dim=0)],
-            # features = [colors],
-            names=[torch.stack(names, dim=0).unsqueeze(-1)],
-        )
+        if instances:
+            bounds, names = zip(
+                *[(v.bounds, v.category_id) for v in self.get_instances()]
+            )
+            detected_boxes = BBoxes3D(
+                bounds=[torch.stack(bounds, dim=0)],
+                # features = [colors],
```

Comment:
dead code?

Reply:
should we be passing color?

Reply:
At some point we can color the boxes according to class, but that's not implemented yet :)

Reply:
This is sort of a placeholder for that

Reply:
I'll just change the comment